validator/validators/performance/performance_validator.go

```diff
@@ -0,0 +1,59 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Here is how the performance declare the validator through three stages at declared in the PR:
* [GenerateLoad](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/validator/models/validator_factory.go#L10): 
* [CheckData](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/validator/models/validator_factory.go#L11) 
* [Cleanup](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/validator/models/validator_factory.go#L12)

However, as stated in the PR, we will only generate load in this PR with [metrics and logs only](https://github.com/aws/amazon-cloudwatch-agent-test/pull/151/files#diff-334efd747575193d3505843e769add29bb2818c73a18ca77c9c4f97105ec3104R35-R39). The get and push validation will be in the upcoming PR.

Reply:
A performance benchmark doesn't really need to check metric values anyways. In my opinion it just generates load and measure the Agent's resource usage (CPU, MEM, DISK, NETIO).

Reply:
Yes that's the point of performance. It does not need to check the statsd's metrics value but only monitor the agent performance.